x-pack/test/security_solution_cypress/cypress/e2e/explore/navigation/navigation.cy.ts

```diff
@@ -113,8 +113,8 @@ describe('top-level navigation common to all pages in the Security app', { tags:
     navigateFromHeaderTo(ENTITY_ANALYTICS);
     cy.url().should('include', ENTITY_ANALYTICS_URL);
   });
-
-  it('navigates to the Kubernetes page', () => {
+  // Skipping this until alternative k8s datasource is found
+  it.skip('navigates to the Kubernetes page', () => {
     navigateFromHeaderTo(KUBERNETES);
     cy.url().should('include', KUBERNETES_URL);
   });
```

Comment:
let's remove dead code/tests, we can always revive them from git history

Reply:
++